### ISSUE TITLE
ValueError traceback raised on ansible-playbook invocation (tested on OSX and Debian)

### ISSUE THREAD

**Issue by yonkeltron** [ansible/ansible#9757]
Hello!
##### Issue Type:

Bug report
##### Ansible Version:

1.7.2 on Debian
1.8.2 on OSX
##### Environment:

On Debian:

```
$ ansible-playbook --version
1.7.2
$ python --version
python 2.7.8
```

On OSX (Yosemite 10.10.1):

```
$ ansible-playbook --version
ansible-playbook 1.8.2
  configured module search path = None
$ python --version
Python 2.7.6
$ uname -a
Darwin Concurrent-Chickpea.local 14.0.0 Darwin Kernel Version 14.0.0: Fri Sep 19 00:26:44 PDT 2014; root:xnu-2782.1.97~2/RELEASE_X86_64 x86_64
```
##### Summary:

Sadly, I received a ValueError traceback when trying to run ansible-playbook:
##### Steps To Reproduce:

Every time I specify an inventory file -- using either `-i` or `--inventory-file` -- I seem to get this error. Very frustrating. Make sure to give a file!
##### Expected Results:

If there's a problem with my invocation of the command, I'd want a message saying what I did wrong. If the error is in a file, it should tell me about the error.
##### Actual Results:

```
$ ansible-playbook -C -c local -i hosts server.yml
Traceback (most recent call last):
  File "/usr/local/Cellar/ansible/1.8.2/libexec/bin/ansible-playbook", line 324, in <module>
    sys.exit(main(sys.argv[1:]))
  File "/usr/local/Cellar/ansible/1.8.2/libexec/bin/ansible-playbook", line 264, in main
    pb.run()
  File "/usr/local/Cellar/ansible/1.8.2/libexec/lib/python2.7/site-packages/ansible/playbook/__init__.py", line 313, in run
    play = Play(self, play_ds, play_basedir, vault_password=self.vault_password)
  File "/usr/local/Cellar/ansible/1.8.2/libexec/lib/python2.7/site-packages/ansible/playbook/play.py", line 165, in __init__
    self._tasks      = self._load_tasks(self._ds.get('tasks', []), load_vars)
  File "/usr/local/Cellar/ansible/1.8.2/libexec/lib/python2.7/site-packages/ansible/playbook/play.py", line 622, in _load_tasks
    (k,v) = t.split("=", 1)
ValueError: need more than 1 value to unpack
```

Please let me know if there is any additional information I can provide to help move things along.

Thanks!

**Comment by abadger** [comment]
Do you know what it is about the inventory file that's causing the error and just want a better error message or is there nothing obviously wrong about the inventory file that's causing this either?

**Comment by yonkeltron** [comment]
I think that ultimately, both are the case. I admit my recent introduction to ansible might have led to me having done something wrong. However, a better error message would be ideal.

AFAICT, there's nothing obviously wrong with the inventory file. Please see all of the ansible files in the project here: https://github.com/SexualHealthInnovations/PrivateResults/tree/master/ansible

**Comment by abadger** [comment]
I found that inventory file or not didn't influence this.  (I have a default inventory file in /etc/ansible/ so that might influence that).  Found that the issue your seeing is because the syntax for includes in server.yml is not correct.  It should be like this:

```
  - include: tasks/util.yml
  - include: tasks/security.yml
```

ie: include takes a single filename, not a list.

If you have other errors that you aren't sure about the ansible-project mailing list might get you a response from people who know more about common pitfalls: https://groups.google.com/forum/#!forum/ansible-project
